wherehows-web/app/components/dataset-compliance.ts

```diff
@@ -219,6 +218,16 @@ export default class DatasetCompliance extends Component {
     return !get(this, 'showGuidedComplianceEditMode');
   });
 
+  /**
+   * Flag indicating if we have compliance available. It's based on whether we receive a compliance info object from the
```

Comment:
This comment is incorrect, if the server returns a 404, we instantiate a new compliance policy on the client, so complianceInfo will always have a value.

Reply:
@mars-lan ps confirm if compliance policy is now available for ***all*** datasets, regardless of `supportedPurgePolicies` for the dataset platform?

Reply:
In that case, should we not instantiate a new `complianceInfo` if it returns 404, and use this as our indicator?

Reply:
Currently 404 means that this dataset does not have a compliance policy because one has never been created for it, which will be the case if it's a new dataset, so `complianceInfo` will not be a property anywhere. Also, not sure why this change is actually needed? Was this a CR from anyone?

Reply:
I see. So the issue is the old method of using `supportedPurgePolicies` is no longer applicable as platforms without that property can still have compliance policies. So if complianceInfo is not a good indicator, I think then we should make it always true.